src/NubankClient/Model/Savings/GetSavingsResponse.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace NubankClient.Model
```

Comment:
@natenho I think that the namespace should represent the file path, I would use `NubankClient.Model.Savings`.

Reply:
You are right! My bad! Fixed